src/main/java/org/glassfish/copyright/AbstractCopyright.java

```diff
@@ -215,8 +219,17 @@ protected void checkCopyright(File file) throws IOException {
 		System.out.println(comment);
 		System.out.println("---");
 	    }
-	    if (c.warn && !c.quiet)
-		warnCopyright(file, r);
+	    if (c.warn && !c.quiet){
+			warnCopyright(file, r);
```

Comment:
Formatting doesn't look right.  Are you using tabs or spaces (preferred)?  Way too much indentation.